.pre-commit-config.yaml

```diff
@@ -37,8 +37,13 @@ repos:
         pass_filenames: false
         language: system
         always_run: true
+-   repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
If I mess with some file and make a commit, prettier is skipped at the moment. I think the files setting might be wrong.

Reply:
Sure, I have update them

Reply:
I don't understand why it didn't run, but setting always_run=true seems to fix it. Can you try again?

Reply:
Well the documentation says: 
files | (optional: default '') the pattern of files to run on.
always_run | (optional: default false) if true this hook will run even if there are no matching files.
https://pre-commit.com/#creating-new-hooks 

So you essentially disabled your previous setting now :smile: 
I guess files should be something like *.css. But as you already excluded file types that should not be formatted, maybe we do not need the files setting.

Reply:
Yes, but I couldn't make it work. I wanted to used the files pattern to restrict it to subdirectories, but somehow it always skipped, assuming that there are no modified files. And since css, scss, js files are only in that directory (or did I miss any?), I guess it should work like this as well.

Reply:
try: `files: ^cpmonitor/static/.*\.(js|ts|css|scss|html)$`
or just use  `files: ^cpmonitor/static/.*$` if you don't want to incorporate the file types
The problem was that files uses regular expressions so for instance a '.' is interpreted as any symbol.

Reply:
Thanks it seems to be working now without always_run